### DISCUSSION TITLE
@scalar/cli - what are the future plans?

### DISCUSSION THREAD

**Discussion by andysm1th** [scalar/scalar#1761] [General]
I noticed in the repos root README.md there is still documentation about the CLI. But when I visit the repo it say's its now 'public archive'.

What is the status of the CLI?

**Comment by antlio** [comment]
hey @andysm1th, the cli use to have it's own repo but has recently been moved to the [scalar repository](https://github.com/scalar/scalar/tree/main/packages/cli#scalar-cli).

**Reply by andysm1th** [reply]
Great thanks a bunch, I missed that

**Reply by antlio** [reply]
as far as the future of CLI is concerned, we've started to sketch out the potential future evolutions, we'd love to hear your feedback if you get to use it — feel free also to reach out if you need any help!

**Reply by gizero** [reply]
> hey @andysm1th, the cli use to have it's own repo but has recently been moved to the [scalar repository](https://github.com/scalar/scalar/tree/main/packages/cli#scalar-cli).

Any update after this @antlio reply? Looks like it's now outdated (link is broken and no more scalar-cli in the `packages/` folder of the scalar repository). I couldn't find cli sources anywhere. Can you please suggest where to look for?

**Reply by antlio** [reply]
pinging @amritk here that might help regarding this!

**Reply by amritk** [reply]
thanks @antlio. hey @gizero the scalar-cli is now in our closed source repo. The current documentation can be found here:

https://guides.scalar.com/scalar/scalar-cli/getting-started
https://guides.scalar.com/scalar/scalar-cli/commands

Let me know if you have any questions!
